src/libs/SearchAutocompleteUtils.ts

```diff
@@ -172,6 +173,8 @@ function filterOutRangesWithCorrectValue(
             return categoryList.get().includes(range.value);
         case CONST.SEARCH.SYNTAX_FILTER_KEYS.TAG:
             return tagList.get().includes(range.value);
+        case CONST.SEARCH.SYNTAX_ROOT_KEYS.GROUP_BY:
+            return groupByList.includes(range.value);
```

Comment:
This highlights the group by syntax on a chat type too, which is unsupported: https://github.com/Expensify/App/issues/75411